specification/gp-connect-access-record-fhir.yaml

```diff
@@ -115,54 +148,52 @@ info:
 
     See[Consumer Supplier Test Assurance for achieving Technical Conformance](https://github.com/nhsconnect/gpc-consumer-support/wiki/Document-library#consumer-supplier-test-assurance-for-achieving-technical-conformance) (PDF) for full details of the technical conformance process and relevant artefacts.
 
     ## Onboarding
 
-    # Expressing an interest
+    ### Expressing an interest
 
     If you meet the prerequisites and have a product that can integrate with GP Connect person facing services, you should express an interest with us by submitting a use case.
 
-    The main purpose of the use case is to help us understand how you plan to use GP Connect person facing services APIs and the business issue you are looking to address. You should email your use case to the GP Connect team.
+    The main purpose of the use case is to help us understand how you plan to use GP Connect person facing services APIs and the business issue you are looking to address. You should email your use case to the GP Connect team.
 
     Your use case should include the following information as a minimum:
     - the business problem you are intending to solve using GP Connect
     - how GP Connect will be used to benefit patients and staff
     - which of the GP Connect products you will use to benefit patients and staff
     - any end-user organisations you are currently working with
-    - who your clinical safety officer is and, where available, your clinical risk management process documentation
+    - who your clinical safety officer is and, where available, your clinical risk management process documentation
 
     Once we receive your use case, we'll respond within 14 days.
 
+    ### Consumer assurance process
 
+    On approval of a use case, we will support you through the assurance process through to go live. We will discuss the assurance process and artefacts with you to help you understand the requirements.
 
-    ## Consumer assurance process
+    Start your development work within 6 months of use case approval. If you miss this date, a review or new submission of the use case will be required. Changes or additional development will also require a review or new use case submission. For full details of the technical conformance process, see GP Connect Consumer Supplier Test Assurance for achieving Technical Conformance (PDF).
 
-    On approval of a use case, we will support you through the assurance process through to go live. We will discuss the assurance process and artefacts with you to help you understand the requirements.
+    ### Clinical assurance process
```

Comment:
Not sure about this clinical assurance section - has this been checked with CSO? We've said explicitly that the API isn't for clinical use, so why is clinical testing required?